src/test/java/easytests/api/v1/controllers/UsersControllerTest.java

```diff
@@ -1,14 +1,113 @@
+    private static String firstName = "firstName";
+    private static String lastName = "lastName";
+    private static String surname = "surname";
+    private static String email = "email";
+    private static String isAdmin = "isAdmin";
+    private static String state = "state";
+
+    @Autowired
+    private MockMvc mvc;
+
+    @MockBean
+    private UsersService usersService;
+
+    @MockBean
+    private UsersOptionsBuilder usersOptionsBuilder;
+
+    private UsersSupport usersSupport = new UsersSupport();
+
+    private SubjectsSupport subjectsSupport = new SubjectsSupport();
+
+    @MockBean
+    private SessionServiceInterface sessionService;
+
+    @Test
+    public void testListSuccess() throws Exception {
+
+        final List<UserModelInterface> usersModels = new ArrayList<>();
+        IntStream.range(0, 2).forEach(idx -> {
+            final UserModel userModel = new UserModel();
+            userModel.map(this.usersSupport.getEntityFixtureMock(idx));
+            usersModels.add(userModel);
+        });
+        when(this.sessionService.getUserModel()).thenReturn(usersModels.get(0));
+        when(this.usersService.findAll()).thenReturn(usersModels);
+
+        mvc.perform(get("/v1/users")
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andExpect(content().contentTypeCompatibleWith(MediaType.APPLICATION_JSON))
+                .andExpect(content().json(new JsonSupport()
+                        .with(new JsonSupport()
+                                .with(id, usersModels.get(0).getId())
+                                .with(firstName, usersModels.get(0).getFirstName())
+                                .with(lastName, usersModels.get(0).getLastName())
+                                .with(surname, usersModels.get(0).getSurname())
+                                .with(email, usersModels.get(0).getEmail())
+                                .with(isAdmin, usersModels.get(0).getIsAdmin())
+                                .with(state, usersModels.get(0).getState()))
+                        .with(new JsonSupport()
+                                .with(id, usersModels.get(1).getId())
+                                .with(firstName, usersModels.get(1).getFirstName())
+                                .with(lastName, usersModels.get(1).getLastName())
+                                .with(surname, usersModels.get(1).getSurname())
+                                .with(email, usersModels.get(1).getEmail())
+                                .with(isAdmin, usersModels.get(1).getIsAdmin())
+                                .with(state, usersModels.get(1).getState()))
+                        .build()
+                ))
+                .andReturn();
+    }
+
+    @Test
+    public void testListForbidden() throws Exception {
+        final UserModelInterface userModel = new UserModel();
+        userModel.map(this.usersSupport.getEntityFixtureMock(1));
```

Comment:
It is hard to understand that user with idx0 is admin and with idx1 is not.
I think that a good idea is:
manually set is admin to be true or false for each case.
Than, if we modify support class data test will not fail.

But there is one more clean case:
 - add two methods into users support class: getAdminUser and getNoneAdmoinUser and act with that users or smt like that.